testkit/src/mill/testkit/IntegrationTester.scala

```diff
@@ -91,7 +91,11 @@ object IntegrationTester {
       )
     }
 
-    def millTestSuiteEnv: Map[String, String] = Map("MILL_TEST_SUITE" -> this.getClass().toString())
+    def millTestSuiteEnv: Map[String, String] =
+      Map("MILL_TEST_SUITE" -> this.getClass().toString()) ++
+        sys.props.get("TEST_SCALA_2_13_VERSION")
+          .map("TEST_SCALA_2_13_VERSION" -> _)
+          .toSeq
```

Comment:
Rather than patching this here, can we duplicate `TEST_SCALA_2_13_VERSION` as a `forkEnv` in `integration/package.mill`? This seems like a Mill-internal concern and shouldn't need to be published for external users of `mill-testkit`

Reply:
Should be fixed

Reply:
looks good, once it's green let's land this